### ISSUE TITLE
Issues w/VM setup after Python3 version updated to 3.8?

### ISSUE THREAD

**Issue by TesterTesterson1004** [microsoft/MCW-Modernizing-Data-Analytics-with-SQL-Server-2019#4]
- When attempting to follow the 'Modernizing Data Analytics with SQL Server 2019 before the hands-on lab setup guide' on a Windows10/1809 freshly-installed VM, the instructs appear to work OK up to the 7th command listed in Step 10:

pip3 install -r https://aka.ms/azdata --user

- When running that cmd I rec'v error 'connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed', for which I believe a fix to bypass cert warning is to use cmd:

pip3 install --trusted-host pypi.org --trusted-host files.pythonhosted.org -r https://aka.ms/azdata --user

- However, after doing so I encounter a series of other errors regarding 'error Microsoft Visual C++ 14.0 is required', and later also 'c1083 cannot open include file sqlfront.h' when attempting to build pymssql.  The former *might* have been eliminated by a manual install of  build tools, the latter I cannot identify how to fix.

NOTE:  I am not a python expert, however a certain amount of searching/troubleshooting makes me wonder if the recent (~10/16/19) release of Python version 38 may have been playing a part in breaking previously-functional setup instructs?  (I have not yet had the time to see if using the same setup doc, but forcing an install of a previous version of Python helps)

**Comment by donnatamta** [comment]
exact same issue here:(

**Comment by timahenning** [comment]
Thank you for reporting this issue.  Please take a moment to review the help and support guidance.  https://microsoftcloudworkshop.com/  We will review your issue soon.

Tim

**Comment by donnatamta** [comment]
I have downgraded Python to 3.6 and Visual Studio to 2017 (I had 2019) and was able to install azdata without errors. Just a workaround until this is fixed.

**Comment by timahenning** [comment]
Thank you for the update.

Tim

**Comment by thevinnie** [comment]
I ran into this same issue and ended up installing the build tools as described in this blog.  That allowed for the azdata installation to complete.

https://www.scivision.dev/python-windows-visual-c-14-required/

Fix the error for Python on Windows:
**error Microsoft Visual C++ 14.0 is required**
as follows.

Visual Studio changed the Build Tools from being C++ specific in late 2017. Thus newer Visual Studio versions work in place of older versions.

- Install using any ONE of these choices:
   Microsoft Build Tools for Visual Studio.
   Alternative link to Microsoft Build Tools for Visual Studio.
   Offline installer: vs_buildtools.exe (I chose this option)
- Select: Workloads → C++ build tools.
- Install options: select only the “Windows 10 SDK” (assuming the computer is Windows 10). To use MSVC cl.exe C / C++ compiler from the command line, additionally select the C++ build tools.
